src/components/confusion/ConfusionBarometer.tsx

```diff
@@ -1,32 +1,13 @@
 import React, { useEffect } from 'react'
```

Comment:
Extracted the chart components from the barometer to be able to reuse them in the evaluation.